src/main/java/io/github/thebusybiscuit/slimefun4/utils/NumberUtils.java

```diff
@@ -171,9 +176,31 @@ private NumberUtils() {}
         }
     }
 
-    public static @Nonnull String getTimeLeft(int seconds) {
+    public static @Nonnull String getTimeLeft(int ticksLeft) {
         String timeleft = "";
 
+        /**
+         * This is the amount calculated in addProgress(). 
+         * This will be the scale used for timer.
+         */
+        double timeScale = (TICK_RATE / 10.0d);
+        double normalScale = ((timeScale) - Math.round(timeScale));
+
+        int seconds = 0;
+
+        //Adjust for positive change
+        if (normalScale < 0) {
+            seconds = (int) (ticksLeft * (normalScale / 2 + 1.0d));
+        }
+        // Adjust for negative change 
+        else if (normalScale > 0) {
+            seconds = (int) (ticksLeft * (normalScale + 1.0d));
+        }
+        // No change 
+        else {
+            seconds = ticksLeft;
+        }
```

Comment:
Please adhere to our code style.

```suggestion
        if (normalScale < 0) {
            //Adjust for positive change
            seconds = (int) (ticksLeft * (normalScale / 2 + 1.0D));
        } else if (normalScale > 0) {
            // Adjust for negative change 
            seconds = (int) (ticksLeft * (normalScale + 1.0D));
        } else {
            // No change 
            seconds = ticksLeft;
        }
```